BHoMAnalytics_UI/CaptureProjectData.xaml.cs

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+using BH.oM.UI;
+
+namespace BH.UI.Analytics
+{
+    /// <summary>
+    /// Interaction logic for CaptureProjectData.xaml
+    /// </summary>
+    public partial class CaptureProjectData : Window
+    {
+        public CaptureProjectData()
+        {
+            InitializeComponent();
+            this.Show();
+        }
+
+        private void Click_ProjectBtn(object sender, EventArgs e)
+        {
+            ProjectInputPanel.Visibility = Visibility.Visible;
+        }
+
+        private void Click_NonProjectBtn(object sender, EventArgs e)
+        {
+            string projectID = "Non-Project work";
```

Comment:
This wording (Non-Project) is good for now. As discussed, in future iterations we would like another step to ask the user what type of non-project work. This would be to avoid laziness 🙃